projects/packages/jetpack-mu-wpcom/src/features/block-theme-previews/modal.jsx

```diff
@@ -0,0 +1,82 @@
+import { Button, Modal } from '@wordpress/components';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { useEffect } from '@wordpress/element';
+import { __, sprintf } from '@wordpress/i18n';
+import React from 'react';
+import { getPreviewedThemeStylesheet } from './utils';
+
+import './modal.scss';
+
+/**
+ * The modal that shows in the Site Editor the first time a user previews a block theme.
+ */
+export default function BlockThemePreviewsModal() {
+	const stylesheet = getPreviewedThemeStylesheet();
+	const theme = useSelect( select => select( 'core' ).getTheme( stylesheet ), [ stylesheet ] );
+
+	const isInSiteEditor = useSelect( select => !! select( 'core/edit-site' ), [] );
+	const isModalVisible = useSelect(
+		// eslint-disable-next-line @typescript-eslint/no-explicit-any
+		select => select( 'automattic/wpcom-block-theme-previews' ).isModalVisible(),
+		[]
+	);
+	const { dismissModal } = useDispatch( 'automattic/wpcom-block-theme-previews' );
+	const { suppressLivePreviewModal } = useDispatch( 'automattic/wpcom-welcome-guide' );
+
+	useEffect( () => {
+		if ( isInSiteEditor ) {
+			// Hide the legacy modal from the WordPress Editing Toolkit plugin.
+			suppressLivePreviewModal?.();
+		}
+	}, [ isInSiteEditor, suppressLivePreviewModal ] );
+
+	if ( ! isInSiteEditor || ! isModalVisible ) {
+		return null;
+	}
+	return (
+		<Modal
+			className="wpcom-block-theme-previews-modal"
+			onRequestClose={ dismissModal }
+			shouldCloseOnClickOutside={ false }
+		>
+			<div className="wpcom-block-theme-previews-modal__content">
+				<div className="wpcom-block-theme-previews-modal__text">
+					<h1 className="wpcom-block-theme-previews-modal__heading">
+						{ sprintf(
+							// translators: %s: theme name
+							__( 'You’re previewing %s', 'jetpack-mu-wpcom' ),
+							theme?.name?.rendered || stylesheet
+						) }
+					</h1>
+					<div className="wpcom-block-theme-previews-modal__description">
+						<p>
+							{ __(
+								'Changes you make in the editor won’t be applied to your site until you activate the theme.',
+								'jetpack-mu-wpcom'
+							) }
+						</p>
+						<p>
+							{ __(
+								'Try customizing your theme styles to get your site looking just right.',
+								'jetpack-mu-wpcom'
+							) }
+						</p>
+					</div>
+					<div className="wpcom-block-theme-previews-modal__actions">
+						<Button variant="primary" onClick={ dismissModal }>
+							{ __( 'Start customizing', 'jetpack-mu-wpcom' ) }
+						</Button>
+					</div>
+				</div>
+				<div className="wpcom-block-theme-previews-modal__video">
+					<video autoPlay loop muted>
+						<source
+							src="https://videos.files.wordpress.com/gTXUlIAB/wpcom-block-theme-previews-modal.mp4"
```

Comment:
@fushar @matt-west 
We've been receiving feedback indicating that users find navigating back (or exiting) from the Live Preview flow challenging.  

Am I correct in understanding that we can address that issue later?🙂 This could, I think, include solutions like adding a more comprehensive tour pekYwv-3uK-p2#updating-the-welcome-modal, displaying a banner to the editor sidebar pekYwv-3uK-p2#adding-a-banner-to-the-editor-sidebar, and enhancing the Gutenberg interface pekYwv-3uK-p2#comment-2689. 

Or should we consider showing the instructions here in the video? What are your thoughts?

Reply:
> displaying a banner to the editor sidebar pekYwv-3uK-p2#adding-a-banner-to-the-editor-sidebar

Yes, I plan to do the above ☝️ in a future PR! The current PR is already big enough 😄 the scope is to replace the old modal.